ocs_ci/ocs/resources/cloud_manager.py

```
@@ -875,3 +887,29 @@ def create_azure_secret(self):
+class GcpSTSClient(GoogleClient):
+    """
+    Implementation of a GCP Client for STS (Workload Identity Federation) clusters.
+
+    Reuses GoogleClient's GCS storage client for ULS operations but provides
+    WIF parameters (project_number, pool_id, provider_id, service_account_email)
+    for creating google-cloud-storage-sts backingstores.
+
+    """
+
+    @config.run_with_provider_context_if_available
+    def __init__(self, full_auth_dict, *args, **kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)